drivers/charger/charger_axp2101.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2025 BayLibre SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _CHARGER_AXP2101_H_
+#define _CHARGER_AXP2101_H_
+
+#define PMU_STATUS1				0x00
+	#define VBUS_GOOD_INDICATION		BIT(5)
+	#define BATFET_STATE			BIT(4)
+	#define BATTERY_PRESENT			BIT(3)
+	#define BATTERY_IN_ACTIVE_MODE		BIT(2)
+	#define THERMAL_REGULATION_STATUS	BIT(1)
+	#define CURRENT_LIMIT_STATE		BIT(0)
+
+#define PMU_STATUS2				0x01
+	#define BATTERY_CURRENT_DIRECTION	GENMASK(6, 5)
+	#define SYSTEM_STATUS_INDICATION	BIT(4)
+	#define VINDPM_STATUS			BIT(3)
+	#define CHARGING_STATUS			GENMASK(2, 0)
+	#define TRICKLE_CHARGE			0x0
+	#define PRE_CHARGE			0x1
+	#define CONSTANT_CURRENT		0x2
+	#define CONSTANT_VOLTAGE		0x3
+	#define CHARGE_DONE			0x4
+	#define NOT_CHARGING			0x5
+
+#define DATA_BUFFER_START			0x04
```

Comment:
The AXP2101 is configured as an MFD, and each function defines only the necessary registers.

https://github.com/zephyrproject-rtos/zephyr/blob/6c989dee10c5795bb85dc59ad87d8898ee7940d0/drivers/regulator/regulator_axp192.c#L34-L53

Can you do something like that?

Reply:
One more.
If it's not referenced anywhere else, it's better to put it in .c.

Reply:
Done ;)